src/pytest_benchmark/utils.py

```diff
@@ -67,9 +72,16 @@ def __repr__(self):
         return "NameWrapper(%s)" % repr(self.target)
 
 
-def get_tag():
-    info = get_commit_info()
-    return '%s_%s%s' % (info['id'], get_current_time(), '_uncommitted-changes' if info['dirty'] else '')
+def get_tag(project_name=None):
+    info = get_commit_info(project_name)
+    parts = []
+    if info['project']:
```

Comment:
@Artimi I plan to remove the project part (to get back the old "brief" naming style from 3.0) in the next release (alpha2). Let me know if you have objections or not.